### ISSUE TITLE
GGM - ENHANCEMENT - Double jump ability

### ISSUE THREAD

**Issue by Kravenwolf** [TheGameCreators/GameGuruRepo#4467]
Currently, there's a bug where if you jump while you're freefalling, you'll do another jump in mid-air. (#3984 , #4466 )

While tackling this issue, I'd like to suggest the dev team includes double jumping as a feature (as seen in DOOM 2016 / DOOM Eternal).

I'd also like to recommend adding a script/behavior to go along with it that would allow the player to get this ability after picking up an item (like "Thruster Boots".)

**Comment by Lemuripest93** [comment]
Yes please! i've been thinking about adding a second game mode down the line to my game which would require more 'arena shooter' type movement, double jump would be phenomenal 👌
